Utils/hdinsight-node-common/src/com/microsoft/azure/arcadia/serverexplore/ArcadiaSparkWorkspaceNode.java

```diff
@@ -50,8 +54,29 @@ protected synchronized void refreshItems() throws AzureCmdException {
         });
     }
 
+    @Override
+    protected void loadActions() {
+        super.loadActions();
+
+        addAction("Launch workspace", new NodeActionListener() {
+            @Override
+            protected void actionPerformed(NodeActionEvent e) throws AzureCmdException {
+                try {
+                    Desktop.getDesktop().browse(createWorkSpaceLink());
```

Comment:
`DefaultLoader.getIdeHelper().openLinkInBrowser()`

And fix `IDEHelperImpl.openLinkInBrowser()` to `BrowserUtil.open()`